xarray/tests/test_variable.py

```diff
@@ -2898,12 +2898,28 @@ def setUp(self):
         self.cat = PandasExtensionArray(pd.Categorical(["a", "b"] * 5))
 
     def check_orthogonal_indexing(self, v):
-        assert np.allclose(v.isel(x=[8, 3], y=[2, 1]), self.d[[8, 3]][:, [2, 1]])
+        result = v.isel(x=[8, 3], y=[2, 1])
+        expected = self.d[[8, 3]][:, [2, 1]]
+        assert np.allclose(result, expected)
+
+    async def check_orthogonal_async_indexing(self, v):
+        result = await v.isel(x=[8, 3], y=[2, 1]).load_async()
+        expected = self.d[[8, 3]][:, [2, 1]]
+        assert np.allclose(result, expected)
```

Comment:
Could combine like this
```suggestion
        async_result = await v.isel(x=[8, 3], y=[2, 1]).load_async()
        sync_result = await v.isel(x=[8, 3], y=[2, 1]).load()
        expected = self.d[[8, 3]][:, [2, 1]]
        assert np.allclose(async_result, expected)
        assert np.allclose(sync_result, expected)
```